lib/controller.rb

```diff
@@ -56,6 +56,9 @@ class SlowFood < Sinatra::Base
 
   get '/menu' do
     @dishes = Dish.all
+    @starter = Dish.all(category: 'starter')
```

Comment:
So you are no longer using `@dishes` in your view, right? Please remove unused variables.